api/server/call_logs.go

```diff
@@ -2,13 +2,20 @@ package server
 
 import (
 	"bytes"
+	"io"
 	"net/http"
 
 	"github.com/fnproject/fn/api"
 	"github.com/fnproject/fn/api/models"
 	"github.com/gin-gonic/gin"
 )
 
+// note: for backward compatibility, will go away later
```

Comment:
i don't think this is really true, we should probably support json perpetually if somebody asks for it